Mage.Tests/src/test/java/org/mage/test/cards/abilities/keywords/BackupTest.java

```diff
@@ -137,4 +137,21 @@ public void MirrorShieldHopliteSourceTest() {
         execute();
         assertPowerToughness(playerA, "Mirror-Shield Hoplite", 4, 4);
     }
+
+    @Test
+    public void StreetwiseNegotiatorTest() {
+        addCard(Zone.BATTLEFIELD, playerA, "Aegis Turtle");
+        addCard(Zone.BATTLEFIELD, playerA, "Forest", 2);
+        addCard(Zone.HAND, playerA, "Streetwise Negotiator", 1);
+
+        castSpell(1, PhaseStep.PRECOMBAT_MAIN, playerA, "Streetwise Negotiator");
+        addTarget(playerA, "Aegis Turtle");
+        // Should deal 6 damage
+        attack(1, playerA, "Aegis Turtle");
+
+        setStrictChooseMode(true);
+        execute();
```

Comment:
huh, I've always seen a stop set, didn't realize it would work without that

Reply:
![image](https://github.com/magefree/mage/assets/34709007/887f57b7-a0eb-416b-aaa8-61dc04a203be)

For reference this is stopping at UPKEEP of turn 2.